tenzir/integration/tests/feather.bats

```diff
@@ -0,0 +1,43 @@
+setup() {
+  bats_load_library bats-support
+  bats_load_library bats-assert
+  bats_load_library bats-tenzir
+}
+
+@test "file roundtrip" {
+  check tenzir "from ${BATS_TENZIR_DATADIR}/inputs/zeek/conn.log.gz read zeek-tsv | top proto"
+  file=$(mktemp)
+  check tenzir "from ${BATS_TENZIR_DATADIR}/inputs/zeek/conn.log.gz read zeek-tsv | to ${file} write feather"
+  check tenzir "from ${file} read feather | top proto"
+}
+
+@test "batch sizes" {
+  venv_dir=$(mktemp -d)
+  python -m venv --system-site-packages "${venv_dir}"
+  . "${venv_dir}/bin/activate"
+  if ! python -c pyarrow; then
+    pip install pyarrow
+  fi
+
+  tenzir "from ${BATS_TENZIR_DATADIR}/inputs/zeek/conn.log.gz read zeek-tsv | batch 512 | write feather" |
+    check python "${BATS_TENZIR_MISCDIR}/scripts/print-arrow-batch-size.py"
+}
+
+@test "invalid format" {
+  check ! tenzir 'version | write json | read feather'
```

Comment:
We should change this test to not use version since version produces different length outputs